app/ui/client/lib/fileUpload.js

```diff
@@ -55,6 +55,14 @@ export const uploadFileWithMessage = async (rid, tmid, { description, fileName,
 			Session.set('uploading', uploads);
 		},
 	});
+	if (Session.get('uploading').length) {
+		UserAction.start(rid, USER_UPLOADING, { tmid });
+	}
+	const msgUploadingInterval = setInterval(function() {
+		if (Session.get('uploading').length) {
+			UserAction.start(rid, USER_UPLOADING, { tmid });
+		}
+	}, 5000);
```

Comment:
Do you have plans to change this to a method inside `UserAction`?

Reply:
Yeah.. that's right..This function can be reusable for uploading and recording features. 
I'll do it shortly